Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM python:3.9.0-alpine3.12
+
+COPY ./requirements.txt ./requirements.txt
+COPY ./config.json ./config.json
+COPY ./export.json ./export.json
+COPY ./credentials.json ./credentials.json
+COPY ./transformer.py ./transformer.py
+
+RUN pip install --upgrade google-api-python-client google-auth-httplib2 google-auth-oauthlib
+RUN pip install -r requirements.txt
```

Comment:
For build cache optimisation this should be just after copying the requirements file.

Reply:
done